### PR TITLE
Added Image & page memory on MemoryTimeLine.

### PR DESCRIPTION
**Pull request by Balajims88**
Related issue #413

### CODE REVIEW

**Comment by Balajims88** [comment]
The branch "stable-memorytimeline" is aimed at adding support for memory timeline (as mentioned in https://webkit.org/blog/6425/memory-debugging-with-web-inspector/) in WPE. There are four different categories of memory statistics that needs to be implemented as discussed in the link. This includes 1.Javascript, 2.Images, 3.Layers, 4.Page.

The earlier submitted issue #413 is to bring in the support for memory timeline in general as well as to bring in the support for javascript. 

This pull request is to bring in support for PAGE AND IMAGE Memory timeLine.

File: Source/WebCore/page/ResourceUsageThread.h [hunk 1]
```diff
@@ -35,6 +35,7 @@
 #include <wtf/Lock.h>
 #include <wtf/NeverDestroyed.h>
 #include <wtf/Noncopyable.h>
+#include <WebCore/MemoryCache.h>
```
Comment (mcatanzaro) [inline Source/WebCore/page/ResourceUsageThread.h]: This needs to be `#include "MemoryCache"`

Comment (Balajims88) [reply Source/WebCore/page/ResourceUsageThread.h]: done

Comment (Balajims88) [reply Source/WebCore/page/ResourceUsageThread.h]: Hi @mcatanzaro,
 Also, I am looking Layers Memory TimeLine on web inspector...
we have to calculate all the Graphics Layer memory for a layer.
so, may I get the memory from the `double RenderLayerBacking::backingStoreMemoryEstimate()` or `CompositingCoordinator` or something else..
can you please suggest me?

Comment (mcatanzaro) [reply Source/WebCore/page/ResourceUsageThread.h]: Sorry, I don't know.

**Pull request merged**